projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.html

```diff
@@ -0,0 +1,262 @@
+        {{ 'Physical description' | translate }}:
+      </dt>
+      <dd class="col-sm-7 col-md-8 mb-0">
+        <ng-container *ngIf="record.metadata.extent">
+          {{ record.metadata.extent }}<ng-container *ngIf="record.metadata.otherMaterialCharacteristics">,</ng-container>
+        </ng-container>
+        <ng-container *ngIf="record.metadata.otherMaterialCharacteristics">
+          {{ record.metadata.otherMaterialCharacteristics }}<ng-container *ngIf="record.metadata.formats">,</ng-container>
+        </ng-container>
+        <ng-container *ngIf="record.metadata.formats">
+          {{record.metadata.formats.join(', ') }}
+        </ng-container>
+      </dd>
+    </ng-container>
+
+    <!-- ADDITIONAL MATERIALS -->
+    <ng-container *ngIf="record.metadata.additionalMaterials">
+      <dt class="col-sm-3 offset-sm-2 offset-md-0">
+        {{ 'Additional Materials' | translate }}:
+      </dt>
+      <dd class="col-sm-7 col-md-8 mb-0">
+        {{ record.metadata.additionalMaterials }}
+      </dd>
+    </ng-container>
+
+    <!-- SERIES -->
+    <ng-container *ngIf="record.metadata.series">
+      <dt class="col-sm-3 offset-sm-2 offset-md-0">
+        {{ 'Series' | translate }}:
+      </dt>
+      <dd class="col-sm-7 col-md-8 mb-0">
+        <ul class="list-unstyled mb-0">
+          <li *ngFor="let serie of record.metadata.series">
+            {{ serie._text }}
+          </li>
+        </ul>
+      </dd>
+    </ng-container>
+
+    <!-- UNIFORM TITLE -->
+    <ng-container *ngIf="record.metadata.titlesProper">
+      <dt class="col-sm-3 offset-sm-2 offset-md-0">
+        {{ 'Uniform title' | translate }}:
+      </dt>
+      <dd class="col-sm-7 col-md-8 mb-0">
+        {{ record.metadata.titlesProper.join('; ') }}
+      </dd>
+    </ng-container>
+
+    <!-- IS PART OF -->
+    <ng-container *ngIf="record.metadata.is_part_of">
+      <dt class="col-sm-3 offset-sm-2 offset-md-0">
+        {{ 'Is part of' | translate }}:
+      </dt>
+      <dd class="col-sm-7 col-md-8 mb-0">
+        {{ record.metadata.is_part_of }}
+      </dd>
+    </ng-container>
+
+    <!-- SUBJECTS -->
+    <ng-container *ngIf="record.metadata.subjects">
+      <dt class="col-sm-3 offset-sm-2 offset-md-0">
+        {{ 'Subjects' | translate }}:
+      </dt>
+      <dd class="col-sm-7 col-md-8 mb-0">
+        {{ record.metadata.subjects.join('; ') }}
+      </dd>
+    </ng-container>
+
+    <!-- NOTES -->
+    <ng-container *ngIf="record.metadata.notes && record.metadata.notes.length > 0">
+      <dt class="col-sm-3 offset-sm-2 offset-md-0">
+        {{ 'Notes' | translate }}:
+      </dt>
+      <dd class="col-sm-7 col-md-8 mb-0">
+        <ul class="list-unstyled mb-0">
+          <li *ngFor="let note of record.metadata.notes" [innerHTML]="note | nl2br"></li>
+        </ul>
+      </dd>
+    </ng-container>
+
+    <!-- IDENTIFIEDBY -->
+    <ng-container *ngIf="identifiedBy as identifiers">
+      <ng-container *ngIf="identifiers.length > 0">
```

Comment:
Cosmetic : Not specialist but it should be condensed into only one IF statement ?

Reply:
This is an optimization to avoid generating data a second time.